front/src/modules/activities/timeline/components/Timeline.tsx

```diff
@@ -132,6 +144,9 @@ export function Timeline({ entity }: { entity: CommentableEntity }) {
             commentThread={commentThread}
           />
         ))}
+        <StyledStartIcon>
+          <IconCircleDot />
```

Comment:
According to the figma, the icon should be 20px by 20px, you can achieve that by using:           <IconCircleDot size={theme.icon.size.lg} />